concert/devices/cameras/dummy.py

```diff
@@ -169,12 +194,134 @@ async def _grab_real(self):
 
         image = await run_in_executor(self._reader.read, self.index)
         self.index += 1
+        await asyncio.sleep((1 / (await self.get_frame_rate())).to(q.s).magnitude)
 
         return image[roi_y0.magnitude:y_region, roi_x0.magnitude:x_region]
 
 
 class BufferedCamera(Camera, base.BufferedMixin):
 
+    async def __ainit__(self, background=None, simulate=True):
+        await Camera.__ainit__(self, background=background, simulate=simulate)
+
+    @transition(target='readout')
+    async def _start_readout_real(self):
+        pass
+
+    @transition(target='standby')
+    async def _stop_readout_real(self):
+        pass
+
     async def _readout_real(self):
         for i in range(3):
             yield await self.grab()
+
+
+class TangoCameraBase(base.RemoteMixin):
+
+    """Tango remote camera base."""
+
+    async def __ainit__(self, tango_device):
+        self._device = tango_device
+
+    async def _get_state(self):
+        return await self._device.get_camera_state()
+
+    async def _record_real(self):
+        await self._device.start_recording()
+
+    async def _stop_real(self):
+        await self._device.stop_recording()
+
+    async def _get_exposure_time(self):
+        return (await self._device['exposure_time']).value * q.s
+
+    async def _set_exposure_time(self, value):
+        await self._device.write_attribute('exposure_time', value.to(q.s).magnitude)
+
+    async def _get_frame_rate(self):
+        return (await self._device['frame_rate']).value / q.s
+
+    async def _set_frame_rate(self, value):
+        await self._device.write_attribute('frame_rate', value.to(1 / q.s).magnitude)
```

Comment:
Thanks for the hint, I am not gonna spoil the myriad of commits for cosmetics like this now, especially since concert is full of 1 / s instead of Hz. But yeah, clever.